datarequest.py

```diff
@@ -36,19 +37,20 @@
            'api_datarequest_data_ready']
 
 
-DRCOLLECTION    = 'home/datarequests-research'
+DRCOLLECTION     = 'home/datarequests-research'
+SCHEMACOLLECTION = 'yoda/datarequest/schemas/youth'
```

Comment:
Use system collection constant for yoda: https://github.com/UtrechtUniversity/irods-ruleset-uu/blob/a519b042ddad4d6e18ca1f95b323823d233ddf9f/util/constants.py#L18